tests/src/whisk/core/cli/test/ApiGwTests.scala

```diff
@@ -80,6 +80,7 @@ class ApiGwTests
       }
       finally {
         val deleteresult = wsk.api.delete(basepathOrApiName = testbasepath)
+        Thread.sleep(5000)  // Test suite will overrun the local dev throttling limit
```

Comment:
It will break eventually. I think it's too brittle of an approach. Should we rather exempt `whisk.system` from throttles?